requirements.txt

```diff
@@ -1,10 +1,10 @@
-numpy >= 1.21.2
-plotly >= 4.12.0
-tabulate >= 0.8.9
-openpyxl >= 3.0.6
-IPython >= 7.22.0
+numpy == 1.21.2
```

Comment:
Lets just limit the one that are raising issues, instead of limiting all of them.

In case user wants to use mario in an environment that some other libraries exist, that can bring lits of issues.

Again, the best practice is to see how to keep everything updated with the latest depency changes, which for us, only pandas is the diffuclt one.

Reply:
I would avoid limiting tabulate, poorly and pyyaml. I fear the others are too crucial for the correct functioning of MARIO. If an expert user comes and sees some conflicts with some packages, I guess he will be able to sort them out. We can't risk the majority of our users (which are not expert programmers for now) can't use Mario because of incompatible dependencies